packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,17 +1,64 @@
 import BigNumber from 'bignumber.js'
 import { Attestations } from '../generated/types/Attestations'
-import { BaseWrapper, proxySend } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend, toBigNumber } from './BaseWrapper'
 
 export interface AttestationStat {
   completed: number
   total: number
 }
 
+export interface AttestationsToken {
+  address: string
+  fee: BigNumber
+}
+
+export interface AttestationsConfig {
+  attestationExpirySeconds: BigNumber
+  attestationRequestFees: AttestationsToken[]
+}
+
+/**
+ * Contract for managing identities
+ */
 export class AttestationsWrapper extends BaseWrapper<Attestations> {
   setAccountDataEncryptionKey = proxySend(
     this.kit,
     this.contract.methods.setAccountDataEncryptionKey
   )
+  attestationExpirySeconds = proxyCall(
+    this.contract.methods.attestationExpirySeconds,
+    undefined,
+    toBigNumber
+  )
+  attestationRequestFees = proxyCall(
+    this.contract.methods.attestationRequestFees,
+    undefined,
+    toBigNumber
+  )
+
+  async getConfig(): Promise<AttestationsConfig> {
+    const events = await this.contract.getPastEvents('AttestationRequestFeeSet', { fromBlock: 0 })
```

Comment:
we are asking the node to go over the whole blockchain history to search for Events. That can be a **really heave workload** for them!

Reply:
Agreed. Feel free to add a list of tokens to Attestations, or just use the address of the only stable token we support right now.

Also, looks like we don't actually set these fees anywhere on our deployed contracts. @nambrot, do you know anything about this? Might make sense to just leave this as a TODO for now.